test/metabase/api/downloads_exports_test.clj

```diff
@@ -0,0 +1,184 @@
+(ns metabase.api.downloads-exports-test
+  "Tests for the various Download and Export features in the app.
+
+  TODO: Add tests to cover the many places data can be exported or downloaded:
+  - Question direct downloads
+  - Unsaved question direct downloads
+  - Dashcard direct downloads
+  - Shared Question downloads
+  - Static Embedding Dashboard/dashcard downloads
+  - Dashboard Subscription Attachments
+  - Alert attachments"
+  (:require
+   [clojure.data.csv :as csv]
+   [clojure.set :as set]
+   [clojure.test :refer :all]
+   [metabase.test :as mt]
+   [metabase.test.data.interface :as tx]))
+
+(def ^:private pivot-base-rows
+  (for [a ["AA" "AB" "AC" "AD"]
+        b ["BA" "BB" "BC" "BD"]
+        c ["CA" "CB" "CC" "CD"]
+        d ["DA" "DB" "DC" "DD"]]
+    [a b c d 1]))
+
+(tx/defdataset pivot-base-data
```

Comment:
Do you really need to create a new dataset for this, would none of the existing test datasets work? The main reason I'm asking is that loading up and syncing test data is pretty slow on some of our Cloud databases, this might take 30-60 seconds for Snowflake versus 0 if we used an existing dataset